weblate/addons/management/commands/install_addon.py

```diff
@@ -65,9 +65,10 @@ def handle(self, *args, **options):
                             )
                     raise CommandError("Invalid addon configuration!")
             addons = Addon.objects.filter_component(component).filter(name=addon.name)
-            if addons.exists():
+            if addons:
                 if options["update"]:
-                    addons.update(configuration=configuration)
+                    for addon_component in addons:
+                        addon_component.configure(configuration)
```

Comment:
Good point, it needs an addon not a model instance:

```suggestion
                        addon_component.addon.configure(configuration)
```